.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,411 @@
+name: Run Tests
```

Comment:
This is pulled out of the staging pipeline, with the addition of the separation of the tests in spec/services.

PRs and the merge queue now run these tests directly instead of the staging pipeline having conditions about whether to do the deployment.